### PR TITLE
Clean up build

### PR DESCRIPTION
**Pull request by basil**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,10 +2,6 @@ target
 work
 .DS_Store
 
-#ignore everything in the mvn directory but the wraper property file
-.mvn
-!.mvn/wrapper/maven-wrapper.properties
-
```
Comment (basil) [inline .gitignore:8]: This repository did not use Maven Wrapper, so this was dead code.

File: pom.xml [hunk 2]
```diff
@@ -6,18 +6,20 @@
     <groupId>org.jenkins-ci</groupId>
     <artifactId>jenkins</artifactId>
     <version>1.87</version>
+    <relativePath />
```
Comment (basil) [inline pom.xml:9]: Consistent with the archetype.

File: pom.xml [hunk 3]
```diff
@@ -6,18 +6,20 @@
...
   </parent>
 
   <groupId>org.jenkins-ci.infra</groupId>
   <artifactId>pipeline-metadata-utils</artifactId>
-  <version>1.6-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
```
Comment (basil) [inline pom.xml:14]: Consistent with the archetype.

File: pom.xml [hunk 4]
```diff
@@ -6,18 +6,20 @@
...
   <name>Pipeline Metadata Utils</name>
   <description>Utilities for use in generating metadata and documentation for Pipeline steps or other Describables</description>
 
   <properties>
-    <gitHubRepo>jenkins-infra/pipeline-metadata-utils</gitHubRepo>
+    <revision>1.6</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <gitHubRepo>jenkins-infra/${project.artifactId}</gitHubRepo>
```
Comment (basil) [inline pom.xml:21]: Consistent with the archetype.

File: pom.xml [hunk 5]
```diff
@@ -6,18 +6,20 @@
...
     <spotbugs.excludeFilterFile>${project.basedir}/src/spotbugs/spotbugs-excludes.xml</spotbugs.excludeFilterFile>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```
Comment (basil) [inline pom.xml:20]: Already part of the parent POM.

File: pom.xml [hunk 6]
```diff
@@ -69,7 +71,6 @@
     <dependency>
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter</artifactId>
-      <version>5.9.0</version>
```
Comment (basil) [inline pom.xml:72]: Already part of the parent POM.

**Review by MarkEWaite — APPROVED** [review]

**Pull request merged**